src/controller/classes/classes.service.ts

```diff
@@ -176,59 +189,60 @@ export class ClassesService {
                 },
             };
             const createdEvent = await calendar.events.insert(eventData);
+
             const saveClassDetails = await db.insert(classesGoogleMeetLink).values({
```

Comment:
can't we store the Google Calendar event ID so we can use it for later operations?